lib/Raven/Processor/SanitizeHttpHeadersProcessor.php

```diff
@@ -67,6 +67,6 @@ public function process(&$data)
      */
     private function getDefaultHeaders()
     {
-        return ['Authorization', 'Proxy-Authorization', 'X-Csrf-Token', 'X-CSRFToken', 'X-XSRF-TOKEN'];
+        return ['Authorization', 'Proxy-Authorization', 'X-Authorization', 'Cookie', 'X-Csrf-Token', 'X-CSRFToken', 'X-XSRF-TOKEN'];
```

Comment:
I don't recall anymore why I added the Cookie header there. Even by looking at the `master` code, my original premise doesn't make much sense. Whatever xD